geoengine/types.py

```diff
@@ -660,6 +660,29 @@ def __repr__(self) -> str:
         return f'{self.name}: {self.measurement}'
 
 
+def literal_raster_data_type(
+    data_type: geoengine_openapi_client.RasterDataType
+) -> Literal['U8', 'U16', 'U32', 'U64', 'I8', 'I16', 'I32', 'I64', 'F32', 'F64']:
+    '''Convert a `RasterDataType` to a literal'''
+
+    data_type_map: dict[
+        geoengine_openapi_client.RasterDataType,
+        Literal['U8', 'U16', 'U32', 'U64', 'I8', 'I16', 'I32', 'I64', 'F32', 'F64']
+    ] = {
+        geoengine_openapi_client.RasterDataType.U8: 'U8',
```

Comment:
can you not just do str(data_type) ?

Reply:
This does not guarantee that it is a literal of exactly these values, at least not for `mypy`.